services/skus/key.go

```diff
@@ -212,3 +215,17 @@ func NewAuthMwr(ks httpsignature.Keystore) func(http.Handler) http.Handler {
 		})
 	}
 }
+
+func validateOrderCvt(ord *model.Order, cvt map[string]string) error {
+	if loc, ok := cvt["location"]; ok && ord.Location.Valid {
+		if ord.Location.String != loc {
```

Comment:
question: do we need to consider case sensitivity when doing the comparison?

Reply:
Based on the knowledge I've got at the moment, I don't think so. Places like our internal systems that might operate with the `location` term, will do so with the hard-coded values, which must be lower case. Anything else would be a programming mistake.

As for other possible places, I am not sure, but would assume that location must be a lower-case string. Not sure if we even allow human interaction with it.